src/categorical_algebra/FinCats.jl

```diff
@@ -239,7 +239,6 @@ end
 FinCatGraph(g::HasGraph) = FreeCatGraph(g)
 
 is_free(::FreeCatGraph) = true
-equations(::FreeCatGraph) = []
```

Comment:
Did you mean to delete this line, and the other one below? I'm surprised this didn't cause an error. Our test coverage of these might be incomplete.